docs/development/renovate-tutorial.md

```diff
@@ -0,0 +1,145 @@
+# Renovate Hands-on-tutorial
+
+## Introduction
+
+We will learn how to run renovate and experience how it works.
+There are Multiple ways to run Renovate, using CLI, self-hosted server or docker,
+but the easiest way to start with is the [Renovate App](https://github.com/apps/renovate).
+
+You can trigger renovate manually from CLI or docker by running it to pick up any changes needed,
+or you can run [Renovate App](https://github.com/apps/renovate) or a host renovate on a server,
+and it will pick up any needed changes every few minutes/hours.
+
+## Tutorial
+
+- We will work with Github for this tutorial since our app is hosted on Github, open your Github repository
```

Comment:
"open your GitHub repository" does not make sense here. I guess you meant "open up github.com" but why do even that if on the next step you give them a link to click on anyway? You could say make sure you're logged into github?